src/commands/moderation/roleassignment.ts

```diff
@@ -27,7 +27,7 @@ const MEDIA_ANNOUNCEMENT_EMBED = makeEmbed({
 export const roleassignment: CommandDefinition = {
     name: 'roleassignment',
     description: 'Sends the role assignment messages',
-    requiredPermissions: ['BanMembers'],
+    requirements: { permissions: ['BanMembers'] },
```

Comment:
Not a comment on the PR, but @benw202 : Does this make sense, should we take the opportunity to change this to ADMIN & MODERATOR roles?